apps/storybook-react/stories/Shadows.stories.tsx

```diff
@@ -24,7 +24,7 @@ const ShadowBox: React.FC<ShadowBoxProps> = ({
 );
 
 const meta = {
-  title: 'Design Tokens/Shadows',
+  title: 'Tailwind Classes/Shadows',
```

Comment:
I was considering removing this story as Shadows are covered in the design tokens stories but I think it might be a good place to start creating Tailwind specific recipes and examples. Open to thoughts and ideas on this